gapic/src/main/com/google/gapid/perfetto/models/SliceTrack.java

```diff
@@ -242,6 +309,43 @@ public ThreadInfo getThread() {
         return thread;
       }
     }
+
+    public static class GpuSlice extends Slice {
+      private final RenderStageInfo renderStageInfo;
+      public GpuSlice(Row row, ArgSet args) {
+        super(row, args);
+        renderStageInfo = new RenderStageInfo(row.getLong(9), row.getString(10), row.getLong(11),
+            row.getString(12), row.getLong(13), row.getString(14));
+      }
+
+      @Override
+      public String getTitle() {
+        return "GPU Render Stages";
+      }
+
+      @Override
+      public RenderStageInfo getRenderStageInfo() {
+        return renderStageInfo;
+      }
+    }
+  }
+
+  public static class RenderStageInfo {
```

Comment:
I know you've been going the rounds about this name, so ignore this if you are tired of renaming this:

I don't like the name "Render Stages". It's not standard. We should call it `VulkanInfo` or maybe `ApiInfo`. Definitely change the label in the UI to not mention "Render Stages".

Reply:
Changed it to Vulkan Info in the UI for now.. Leaving the class name as is and can be changed later.